examples/guestbook-go/README.md

```diff
@@ -4,7 +4,7 @@ This example shows how to build a simple multi-tier web application using Kubern
 
 ### Step Zero: Prerequisites
 
-This example assumes that you have forked the repository and [turned up a Kubernetes cluster](../../docs/getting-started-guides):
+This example assumes that you have forked the repository and [turned up a Kubernetes cluster](/docs/getting-started-guides):
```

Comment:
Should this instead say (because i successfully ran this example without a fork):
"This example assumes that you have a Kubernetes cluster up and running, either from:
 - Forking the Kubernetes repository and running the "hack/dev-build-and-up.sh" script  
           *script example (dev-build-and-up.sh) goes here*
 - Using a [Getting Started Guide](/docs/getting-started-guides)"